tidy3d/web/api/container.py

```diff
@@ -692,7 +702,7 @@ def pbar_description(task_name: str, status: str) -> str:
 
             return description
 
-        run_statuses = [
+        run_statuses = (
```

Comment:
should keep the type as list since the index do sth matter and showed to user.

Reply:
modified